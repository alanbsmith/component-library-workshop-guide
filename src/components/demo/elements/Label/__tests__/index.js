import { generateSnapshots } from "__tests__/helpers";

import Label from '../index';

generateSnapshots(Label, { htmlFor: 'test' });
