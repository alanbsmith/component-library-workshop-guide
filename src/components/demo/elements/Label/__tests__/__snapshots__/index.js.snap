// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label renders correctly 1`] = `
.c0 {
  color: #636d7f;
  display: inline-block;
  font-size: 0.75rem;
  font-weight: 600;
  line-height: 1.2em;
  text-transform: capitalize;
}

<label
  className="c0"
  htmlFor="test"
/>
`;
